doc/source/whatsnew/v0.21.0.txt

```diff
@@ -38,6 +38,7 @@ Other Enhancements
 - :func:`read_feather` has gained the ``nthreads`` parameter for multi-threaded operations (:issue:`16359`)
 - :func:`DataFrame.clip()` and :func: `Series.cip()` have gained an inplace argument. (:issue: `15388`)
 - :func:`crosstab` has gained a ``margins_name`` parameter to define the name of the row / column that will contain the totals when margins=True. (:issue:`15972`)
+- The ``na_values`` argument for :func:`read_csv` function now has 'null' (in lower case) as a default value that is interpreted as NaN: (:issue:`16471`)
```

Comment:
read_csv is not the only function that has na_values argument, maybe change to:

The default NaN recognized values has gained 'null' as an additional default value,  see in docs description of na_values argument for any of the following functions: `read_csv`, `read_table`, `read_excel`, or `read_fwf`